samples/dependency-injection/structuremap/StructureMap_7/Sample/Program.cs

```diff
@@ -12,25 +12,23 @@ static async Task Main()
         #region ContainerConfiguration
 
         var endpointConfiguration = new EndpointConfiguration("Samples.StructureMap");
-        var container = new Container(
-            action: expression =>
-            {
-                expression.For<MyService>()
-                    .Use(new MyService());
-            });
-        endpointConfiguration.UseContainer<StructureMapBuilder>(
-            customizations: customizations =>
-            {
-                customizations.ExistingContainer(container);
-            });
+
+        var registry = new Registry();
+
+        registry.For<MyService>()
+                  .Use(new MyService());
 
         #endregion
 
         endpointConfiguration.UsePersistence<LearningPersistence>();
         endpointConfiguration.UseTransport<LearningTransport>();
 
-        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+        var endpoint = ServiceCollectionEndpoint.Create(endpointConfiguration,
```

Comment:
@timbussmann @danielmarbach like we discussed yesterday. With a little wrapper we can make it very nice for users to switch to the MS DI adapters instead, what do you think?